### PR TITLE
Trev8939/open mobile scene package

### PR DESCRIPTION
**Pull request by TADraeseke**
For your review when you have a moment @LDuncAndroid. Thanks!

### CODE REVIEW

File: java/open-mobile-scene-package/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Open Scene Map Package
```
Comment (LDuncAndroid) [inline java/open-mobile-scene-package/README.md]: Should this be sentance case or are titles exempt?

File: java/open-mobile-scene-package/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.esri.arcgisruntime.sample.openmobilescenepackage">
+
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+
```
Comment (LDuncAndroid) [inline java/open-mobile-scene-package/src/main/AndroidManifest.xml]: Should we have the OpenGL `<uses-feature>` statement here?

File: java/open-mobile-scene-package/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+* MobileScenePackage
+* SceneView
+
+## Offline data
+1. Download the data from [ArcGIS Online](https://www.arcgis.com/home/item.html?id=7dd2f97bb007466ea939160d0de96a9d).
+1. Extract the contents of the downloaded zip file to disk.
+1. Open your command prompt and navigate to the folder where you extracted the contents of the data from step 1.
+1. Execute the following command:
+`adb push philadelphia.mspk /ArcGIS/Samples/ScenePackage/philadelphia.mspk`
```
Comment (LDuncAndroid) [inline java/open-mobile-scene-package/README.md]: Should this be `adb push philadelphia.mspk <sdcard>/ArcGIS/Samples/ScenePackage/philadelphia.mspk`

File: java/open-mobile-scene-package/README.md (hunk 1 above)
Comment (TADraeseke) [reply java/open-mobile-scene-package/README.md]: You're right...also the content of the title is non-sensical...long day...

File: java/open-mobile-scene-package/src/main/AndroidManifest.xml (hunk 2 above)
Comment (TADraeseke) [reply java/open-mobile-scene-package/src/main/AndroidManifest.xml]: good idea

**Comment by TADraeseke** [comment]
Thanks for the quick review @LDuncAndroid - Those changes are in now!

**Pull request merged**